docs/tutorials/setting-up-python-pycharm.md

```diff
@@ -198,3 +185,84 @@ typing `m.` and pycharm will suggest all possible methods and properties.
 Choose one. Now you can put your cursor inside the property and press F1 to
 see the docs. Or press cmd-down arrow to look inside the library where this
 property is defined. Neat huh? Happy coding.
+
+## Additional features for the PyCharm Professional version
+
+The Professional version of PyCharm (which can be obtained for free for educational institutions and OSS projects) has a few more features to make your development experience much smoother. If you are a student or teacher, register your valid university email address on the [JetBrains student application page](https://www.jetbrains.com/student/) to apply for the free version of PyCharm professional. If you are an open source developer, you can visit the [JetBrains Open Source license page](https://www.jetbrains.com/buy/opensource/) to see if you qualify for a free version of PyCharm professional.
```

Comment:
Again, saying that this can be obtained for free for OSS projects is a bit of a misnomer.